en/frames_plane/reptile_dragon_2.md

```diff
@@ -0,0 +1,391 @@
+
+:::note
+[DSHOT ESC](../peripherals/dshot.md#wiring-connections) were used (not PWM as for the servos).
+To make efficient use of the [DSHOT output port restrictions](../peripherals/dshot.md#wiring-connections), the two ESCs must be wired to FMU output channels 3 and 4.
+:::
+
+### Airspeed Sensor & Pitot Tube
+
+The airspeed sensor was connected to the I2C port on the Pixhawk 5x carrier board with the supplied JST GH I2C cable.
+
+![RD2 pitot plug](../../assets/airframes/fw/reptile_dragon_2/pitot_plug.jpg)
+
+The pitot tube was pushed through the pitot tube mount and then installed in the front fpv camera cut out.
+
+The pitot/static hoses were cut to length and installed to connect the pitot static probe to the airspeed sensor.
+Finally, the pitot static sensor was taped to the sidewall of the airframe (using double sided tape).
+
+### ELRS RX
+
+A custom cable was made to connect the ELRS RX to the JST GH `TELEM2` port of the Holybro Pixhawk 5x.
+
+![ExpressLRS to telem port cable](../../assets/airframes/fw/reptile_dragon_2/elrs_cable.jpg)
+
+The other end of the cable was terminated to a Dupont connector to connect to the standard spaced headers on the ELRS RX.
+The ELRS RX was connected to the cable, and then heatshrink was used to secure the two together. 
+
+![ExpressLRS RX installed in the RD2 airframe](../../assets/airframes/fw/reptile_dragon_2/elrs_pitotstatic.jpg)
+
+A thin radio antenna tube was pushed through the top of the airframe used to mount one of the two ELRS diversity antennas upright.
+The second diversity antenna was taped to the sidewall of the airframe, 90 degrees from the alignment of the first antenna.
+The ELRS RX was attached to the sidewall of the airframe next to the airspeed pressure sensor, using double-sided tape.
+
+### USB
+
+A right angle USB C extension cable was used to allow easy access to the USB C port on the FMU.
+
+![Rear USB cable hatch](../../assets/airframes/fw/reptile_dragon_2/usb_hatch.jpg)
+
+The cable was installed such that it escapes the pixhawk heading towards the aft of the airplane. The cable continues to run to the rear hatch, where the excess length can be securely wound into a knot.
+Access to this cable can be accomplished by simply removing the rear hatch and unknotting the cable.
+
+## Software Setup
+
+For this build, we need to use the PX4 modules [crsf_rc](../modules/modules_driver.md#crsf-rc) and [msp_osd](../modules/modules_driver.md#msp-osd).
+These require some custom configuration to enable.
+
+First, follow [this guide to setup a development environment](../dev_setup/dev_env.md ) and [this guide to get the PX4 source code](../dev_setup/building_px4.md).
+
+Once a build environment has been setup, open a terminal and `cd` into the `PX4-Autopilot` directory.
+To launch the [PX4 board config tool](../hardware/porting_guide_config.md#px4-menuconfig-setup) run:

```

Comment:
Is this the correct link for the tool? i.e. is menuconfig the tool you mean? I always used to edit files if I needed to enable things and haven't played with it since these changes went in.

Reply:
```suggestion
To launch the [PX4 board config tool (`menuconfig`)](../hardware/porting_guide_config.md#px4-menuconfig-setup) run:

```